src/pages/CollectionsListPage/index.js

```diff
@@ -0,0 +1,80 @@
+import React, { useEffect, useState } from "react";
+import { useParams, Link } from "react-router-dom";
+import { useSelector, useDispatch } from "react-redux";
+import { selectUserCollections } from "../../store/collectionsList/selectors";
+import {
+  addNewCollection,
+  fetchUserCollections,
+  setUserCollections,
+} from "../../store/collectionsList/actions";
+import { selectToken } from "../../store/user/selectors";
+import { selectUser } from "../../store/user/selectors";
+
+import Form from "react-bootstrap/Form";
+import Container from "react-bootstrap/Container";
+import Button from "react-bootstrap/Button";
+import { Col } from "react-bootstrap";
+import "./index.css";
+
+export default function CollectionsListPage() {
+  const dispatch = useDispatch();
+  let { id } = useParams();
+  const [collectionName, setCollectionName] = useState("");
+  const token = useSelector(selectToken);
+  const userCollections = useSelector(selectUserCollections);
+  const user = useSelector(selectUser);
+
+  useEffect(() => {
+    dispatch(fetchUserCollections(id));
+  }, [dispatch, setUserCollections]);
+
+  function submitForm(event) {
+    event.preventDefault();
+    dispatch(addNewCollection(collectionName));
+
+    setCollectionName("");
+  }
+
+  return (
+    <>
+      <h1 className="userName">
+        {userCollections[0]?.user.name || user.name}'s collections
+      </h1>
+      <div className="collectionsContainer">
+        {userCollections?.map((item) => (
```

Comment:
nice work using the new optional chaining stuff! it looks cool! only so you are aware, this feature isn't supported in every browser yet cause its pretty new, https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining. Shouldn't be a problem unless its a really old browser

Reply:
ok, good to know that, I didn't suppose optional chaining isn't supported in every browser